recipe/meta.yaml

```diff
@@ -37,25 +39,27 @@ requirements:
     - setuptools >=18.5
 
 # generally failing
-{% set nose_exclude = "recursion" %}
+{% set nose_exclude = "recursion|test_set_matplotlib_formats|test_set_matplotlib_close" %}
```

Comment:
actually, we could probably put these back, as it will nicely skip them if matplotlib is missing... can be another PR...